colorbleed/plugins/maya/publish/extract_yeti_cache.py

```diff
@@ -37,16 +37,23 @@ def process(self, instance):
         if preroll > 0:
             start_frame -= preroll
 
+        kwargs = {}
+        samples = instance.data.get("samples", 0)
+        if samples == 0:
+            kwargs.update({"sampleTimes": "0.0 1.0"})
```

Comment:
If this is redundant (if `"sampleTimes": "0.0 1.0"` is the same as `"samples": 0`) then this kwargs could be removed and just `samples` can be passed to the `pgYetiCommand` - less code and more readable. :)

Reply:
@aardschok did you ever figure out whether [this change](https://github.com/Colorbleed/colorbleed-config/pull/152#discussion_r215656795) would actually behave the same? If so let's simplify this code and avoid the _if..else_ statement.